src/Microsoft.Extensions.PlatformAbstractions/RuntimeEnvironmentExtensions.cs

```diff
@@ -7,15 +7,15 @@ public static class RuntimeEnvironmentExtensions
         private static readonly string OverrideEnvironmentVariableName = "DOTNET_RUNTIME_ID";
         private static readonly string LegacyOverrideEnvironmentVariableName = "DNX_RUNTIME_ID";
```

Comment:
I am not even sure if `DOTNET_RUNTIME_ID` works anymore end to end. I tried it on Debian some time ago and it did not help (modifying /etc/os_release did help)